core/src/miner/mem_pool.rs

```diff
@@ -983,42 +983,6 @@ pub mod test {
         rlp_encode_and_decode_test!(TxOrigin::External);
     }
 
-    #[test]
```

Comment:
What about testing the conversion path: `VerifiedTransaction` -> `Bytes` -> `UnverifiedTransaction` -> `VerifiedTransaction` ?

Reply:
I modified the tests.